datamade/just-spaces
# Update deploy key to fix Travis deploys

## Overview

Update the encrypted deploy key in `.travis.yml` to get deployments working again.

## Notes

The root problem here is that the `travis` CLI can communicate with multiple Travis API servers (travis-ci.com and travis-ci.org). You can pass in a flag specifying which server you want to communicate with, `--pro` (i.e. `.com`) or `--org` (i.e. `.org`). Specifying the `.com` server worked:

```
$ travis setup codedeploy --pro -f -r datamade/just-spaces
```

## Testing instructions

* See the branch build and confirm that it deployed: https://www.travis-ci.com/datamade/just-spaces/builds/103721667
